admin/client/src/styles/_variables.scss

```diff
@@ -697,3 +699,36 @@ $cms-panel-xs: 60px;
 $cms-panel-sm: 300px;
 $cms-panel-md: 448px;
 $cms-panel-lg: 638px;
+
+// Shadows, these need work to make consistent (used in AssetAdmin)
+$shadow-light: rgba(0, 0, 0, .2);
+$dropdown-shadow: 0 0 3px $shadow-light;
+$gallery-item-shadow: 0 1px 4px $gray-light;
+
+// Masks: typically behind modals, drag drop areas
+$page-mask-light: rgba(236, 239, 241, .9);
+
+
+// Variables for use within but not limited to the AssetAdmin module
+// Gallery item path to framework
+
+$framework-path:     "../../../../framework";
```

Comment:
yup I saw that and removed that line, it was originally a straight copy from assetadmin but I've moved that line back.